src/app/components/layouts/DefaultLayout/DefaultLayout.tsx

```diff
@@ -24,9 +27,18 @@ const MenuWrap = styled.div`
   justify-content: space-between;
   width: 100%;
 `
+const BreadCrumbWrap = styled.div`
```

Comment:
This could not be solved with `<Row>` + `<Column>`?

Reply:
No, this css is for the centering on the page (like <main> does)

Reply:
Ok, let's move `BreadCrumbWrap` into its own file, then. Not cluttering `DefaultLayout` with it.